apollo-router/src/services/execution_service.rs

```diff
@@ -216,13 +219,25 @@ impl ExecutionService {
         tracing::debug_span!("format_response").in_scope(|| {
             let mut paths = Vec::new();
             if let Some(filtered_query) = query.filtered_query.as_ref() {
+                let unauthorized_paths = query.unauthorized_paths.iter().map(|path| path.to_string()).collect::<Vec<_>>();
+                if !unauthorized_paths.is_empty() {
+                    event!(Level::ERROR, unauthorized_query_paths = ?unauthorized_paths, "Authorization error",);
+                }
+
                 paths = filtered_query.format_response(
                     &mut response,
                     operation_name,
                     variables.clone(),
                     schema.api_schema(),
                     variables_set,
                 );
+
```

Comment:
if a subgraph sets an error at a path of an unauthorized field (ex: that field is used for joins between subgraphs), that error should be removed from the response